example/src/containers/MetaMaskConnector/MetaMaskConnector.tsx

```diff
@@ -13,6 +14,10 @@ export const MetaMaskConnector = () => {
        if(!isInitiated) {
            alert("Failed to install snap");
        } else {
+           const dotAssetAdded = await addDotAsset();
```

Comment:
@mpetrunic not sure is this right spot in code to add asset ?

Reply:
Will this trigger some user prompt?

Reply:
@mpetrunic It won't trigger prompt.